7th_homework/exer36.py

```diff
@@ -0,0 +1,7 @@
+operation = lambda x, y: x * y
+def print_operation_table(function, num_rows, num_columns):
+    for i in range(1,num_columns + 1):
+        for j in range(1,num_rows + 1):
+            print(function(i,j), end= " ")
+        print()
+print_operation_table(operation, num_rows=6, num_columns=6)
```

Comment:
По условиям, ваша функция должна принимать 1 обязательных аргумент и 2 необязательных, но у  вас она принимает 3 обязательных. Если не хотите менять количество строк и столбцов, то передавать их не нужно.
```suggestion
def print_operation_table(function, num_rows=6, num_columns=6):
    for i in range(1, num_columns + 1):
        for j in range(1, num_rows + 1):
            print(function(i, j), end=" ")
        print()


print_operation_table(operation)
```